rotkehlchen/globaldb/handler.py

```diff
@@ -326,6 +329,7 @@ def search_assets(filter_query: 'AssetsFilterQuery', db: 'DBHandler') -> List[Di
                             'identifier': resolved_eth.identifier,
                             'name': resolved_eth.name,
                             'symbol': resolved_eth.symbol,
+                            'chain': None,
```

Comment:
as per my comment in the docs this canjust be ommitted if this is what the frontend needs